LBH.AdultSocialCare.Api/V1/Infrastructure/SeedConfiguration/UserSeed.cs

```diff
@@ -0,0 +1,38 @@
+using LBH.AdultSocialCare.Api.V1.Infrastructure.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LBH.AdultSocialCare.Api.V1.Infrastructure.SeedConfiguration
+{
+    public class UserSeed : IEntityTypeConfiguration<Users>
+    {
+        public void Configure(EntityTypeBuilder<Users> builder)
+        {
+            builder.HasData(new Users
+            {
+                Id = Guid.NewGuid(),
```

Comment:
We would like to know the GUID Id to use it. And this would also create a new user seed on every migration. Replace with some Guid values like `"6ec90ab3-0e7e-429c-9272-d4773bf53263"`